src/lifecycles/mount.js

```diff
@@ -1,7 +1,7 @@
 import { NOT_MOUNTED, MOUNTED, SKIP_BECAUSE_BROKEN } from '../applications/app.helpers.js';
 import { handleAppError, transformErr } from '../applications/app-errors.js';
 import { reasonableTime } from '../applications/timeouts.js';
-import CustomEvent from 'custom-event';
+const CustomEvent = require('custom-event');
```

Comment:
why cjs here?

Reply:
my (very basic understanding) is this: https://github.com/rollup/rollup/issues/578

Reply:
https://github.com/rollup/rollup-plugin-commonjs

Reply:
Yup, that was the other option. I'll add it in then :)